requirements_all.txt

```diff
@@ -34,7 +34,7 @@ PyRMVtransport==0.3.3
 PySocks==1.7.1
 
 # homeassistant.components.switchbot
-# PySwitchbot==0.13.3
+# PySwitchbot==0.14.0
```

Comment:
Do we still need to comment the requirement?

https://github.com/home-assistant/core/blob/6ec6f0a835b9f24ae24b73e966d0036d7975ee39/script/gen_requirements_all.py#L33

Reply:
Hi @MartinHjelmare,

I'm fairly sure we don't need to. There are other integrations that make use of bleak.